rackspace/src/main/java/org/jclouds/examples/rackspace/cloudfiles/GenerateTempURL.java

```diff
@@ -105,6 +109,19 @@ private void createContainer() throws IOException {
       }
    }
 
+   private void updatePrivateURLKey() throws IOException {
+      // The key can be changed to quickly expire all existing signed URLs.
+      AccountApi accountApi = blobStore
+            .getContext()
```

Comment:
> I wanted it to be more explicit, though.

Ah, OK. In that case, would it make sense to replace the other occurrences of `blobStoreContext` in the example (e.g. `HttpRequest request = blobStoreContext.getSigner(REGION)...`) with `blobStore.getContext()`?

From a user's perspective, I wonder if it's clear whether they can/should be used interchangeably, or when to use which one..?